### PR TITLE
Update/gitlab integration page with a troubleshoot section

### PR DESCRIPTION
**Pull request by amychen1776**
This is geared toward adding in a troubleshooting section for the GitLab page since this integration is very different from the Github one. I added in two issues that I noticed that has come up.

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 1]
```diff
@@ -62,4 +62,11 @@ You can click the Authenticate button to begin the process, or follow the instru
 
 To connect your personal GitLab account, navigate to your [User Profile](https://cloud.getdbt.com/#/profile/) select the [Integrations section](https://cloud.getdbt.com/#/profile/integrations/) from the left sidebar. Under "GitLab", if your account is already connected, you’ll see "Your user account is linked to a GitLab account with username &lt;your-gitlab-username&gt;."
 
-If your account is not connected, you’ll see "This account is not linked to a GitLab account.” Click the button to begin the setup process. You’ll be redirected to GitLab, and then back into dbt Cloud. When you are redirected to dbt Cloud, you should now see your connected account. 
\ No newline at end of file
+If your account is not connected, you’ll see "This account is not linked to a GitLab account.” Click the button to begin the setup process. You’ll be redirected to GitLab, and then back into dbt Cloud. When you are redirected to dbt Cloud, you should now see your connected account. 
+
+## Troubleshooting
+
+### Respositories Errors
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]:
```suggestion
### Errors importing a repository
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 2]
```diff
@@ -62,4 +62,11 @@ You can click the Authenticate button to begin the process, or follow the instru
...
+If you do not see your repository listed, double-check that:
+- You are a maintainer on that repository (only users with a maintainer or owner permissions can set up repository connections
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: I love that you added this point, but I think the failure mode is actually different. You will see the repo, but you should get an error when dbt Cloud attempts to create a Deploy Token in the repo. The repo should show up though

**Review by drewbanin — COMMENTED** [review]
Couple of real quick comments/suggestions. Otherwise looks good!

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md (hunk 2 above)
Comment (amychen1776) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: ah okay, updating it now.

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 3]
```diff
@@ -68,5 +68,8 @@ If your account is not connected, you’ll see "This account is not linked to a
 
 ### Errors importing a repository
 If you do not see your repository listed, double-check that:
-- You are a maintainer on that repository (only users with a maintainer or owner permissions can set up repository connections
 - Your repository is in a Gitlab group you have access to
+
+If you do see your repository listed but after selecting it, it's stuck loading/errors, double-check that:
+- You are a maintainer on that repository (only users with a maintainer or owner permissions can set up repository connections
```
Comment (alexbethea22) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: missing a ")" at the end of this line. 

Other than that, looks great to me!

Comment (amychen1776) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: That's so weird...I see it?

Comment (amychen1776) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: I just fixed it so it's possible you were on an older page?

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 4]
```diff
@@ -62,4 +62,14 @@ You can click the Authenticate button to begin the process, or follow the instru
...
\ No newline at end of file
+If your account is not connected, you’ll see "This account is not linked to a GitLab account.” Click the button to begin the setup process. You’ll be redirected to GitLab, and then back into dbt Cloud. When you are redirected to dbt Cloud, you should now see your connected account. 
+
+## Troubleshooting
+
+### Errors importing a repository
+If you do not see your repository listed, double-check that:
+- Your repository is in a Gitlab group you have access to
+
+If you do see your repository listed but after selecting it, it's stuck loading/errors, double-check that:
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]:
```suggestion
If you do see your repository listed but are unable to import the repository successfully, double-check that:
```

**Review by drewbanin — COMMENTED** [review]
Ok - one more suggestion and then approved on my end.

Thanks Amy!

**Comment by amychen1776** [comment]
> Ok - one more suggestion and then approved on my end.
> 


Works for me! It's hard to write about an error that you didn't see yourself 😓

**Comment by amychen1776** [comment]
@drewbanin I think I need approval from you :)

**Review by drewbanin — APPROVED** [review]
┏┓
┃┃╱╲ In this
┃╱╱╲╲ house
╱╱╭╮╲╲ we merge
▔▏┗┛▕▔ helpful docs PRs
╱▔▔▔▔▔▔▔▔▔▔╲
╱╱┏┳┓╭╮┏┳┓ ╲╲
▔▏┗┻┛┃┃┗┻┛▕▔

**Pull request merged**